manifests/plugin/pulp.pp

```diff
@@ -25,6 +25,10 @@
 #
 # $pulp3_mirror::       Whether this pulp3 plugin acts as a mirror or another pulp node. A pulp3 mirror is the pulp3 equivalent of a pulpnode.
 #
+# $pulp3_api_url::      The URL to the Pulp 3 API
+#
+# $pulp3_content_url::  The URL to the Pulp 3 content
```

Comment:
We've already used this convention in a few places, but in others we've identified that the use of `pulp3` may come back to bite us and used `pulpcore`.

Reply:
I don't mind renaming. Most of these predate the pulpcore convention. In https://github.com/theforeman/smart_proxy_pulp/ it's also the pulp3 module.

Reply:
Let's merge this and I'll raise an issue here to discuss further.